docs/guides/monitor/lamp-stack.md

```diff
@@ -213,6 +224,12 @@ metrics will help you keep tabs on the performance and availability of your web
 services. The per-second metrics granularity means you have the most accurate information possible for troubleshooting
 any LAMP-related issues.
 
+Another powerful way to monitor the availability of a LAMP stack is the [`httpcheck`
+collector](https://learn.netdata.cloud/docs/agent/collectors/go.d.plugin/modules/httpcheck), which pings a web server at
+a regular interval and tells you whether if and how quickly it's responding. The best way to use this collector is from
+a separate node from the one running your LAMP stack, which is why we're not covering it here, but it _does_ work in a
+single-node setup. Just don't expect it to tell you if your whole node crashed.
+
```

Comment:
I would add the mention about regex which will be helpful in case php crashes and the website serves the default apache screen.

Reply:
Done.